pkg/identityserver/utils.go

```diff
@@ -80,3 +81,16 @@ func idStrings(entityIDs ...*ttnpb.EntityIdentifiers) []string {
 	}
 	return idStrings
 }
+
+// isLastAdmin determines if the userID provided belongs to the last admin of the tenant. Returns an error if the
+// condition is true.
+func isLastAdmin(ctx context.Context, st store.Store, id string) error {
+	adminList, err := st.ListAdmins(ctx, store.FieldMask{"ids"})
+	if err != nil {
+		return err
+	}
+	if len(adminList) == 1 && adminList[0].Ids.UserId == id {
```

Comment:
I am not very confident in user ID only comparisons. Can we use `proto.Equal` between `ids` instead ?